fastlane/lib/fastlane/actions/docs/frame_screenshots.md

```diff
@@ -135,9 +135,10 @@ These are defined in the `data` key. This is an array with the following keys fo
 | `keyword` | Similar use as in `default`, except that parameter `text` can be used here because it is screenshot specific. |
 | `title` | Similar use as in `default`, except that parameter `text` can be used here because it is screenshot specific. |
 
-#### Framefile `keyword` and `title` parameters
+#### <a name="keyword-and-title-parameters"></a>Framefile `keyword` and `title` parameters
```

Comment:
This is usually autogenerated when deploying the site. See https://docs.fastlane.tools/actions/frameit/#titles-and-background-optional as an example

Reply:
I prefer to have this under control, because we're referring to it within this document.
When this section title will be renamed later (by someone else), and the site automatically generates a new anchor, the existing reference to the changed anchor will no longer work.